<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>深度学习上色</string>
  </property>
  <widget class="QFrame" name="buttonFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>399</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttonLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnLoadImg">
      <property name="text">
       <string>读取图片</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnColorize">
      <property name="text">
       <string>上色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSaveImg">
      <property name="text">
       <string>保存图片</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>btnSaveImg</zorder>
   <zorder>btnColorize</zorder>
   <zorder>btnLoadImg</zorder>
  </widget>
  <widget class="QLabel" name="imgLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
